src/components/DERP/index.jsx

```diff
@@ -19,11 +21,49 @@ function DERP() {
   const [chainId, setChainId] = useState("-");
   const [name, setName] = useState("-");
   const [city, setCity] = useState("-");
-  const [lastAddresesUsed, set_lastAddresesUsed] = useState([]);
+  const [lastAddressesUsed, set_lastAddressesUsed] = useState([]);
   const [coordinates, setCoordinates] = useState({
     long: undefined,
     lat: undefined,
   });
+  const [ethCallData, setEthCallData] = useState("");
+  const [addresses, setAddresses] = useState([])
+
+  useEffect(() => {
+    if (ethCallData) {
+      return;
+    }
+    // 0xf0002ea9 is the 4 byte signature of balances(address[],address[]).
+    // https://www.4byte.directory/signatures/?bytes4_signature=0xf0002ea9
+    const foundEthCall = log.find(
+      (entry) =>
+        entry.method === "eth_call" &&
+        entry.params?.at(0)?.data?.startsWith("0xf0002ea9")
```

Comment:
Why not use `  const getAndParseDataFromEntry = (entry) => {` which you already have?

Reply:
Because it depends on `eth_getBalance`, which does not get called on my end as we saw last time